src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -52,12 +52,21 @@ public ResponseEntity<Void> deleteTelevision(@PathVariable int id) {
     }
 
     @PutMapping("/{id}/cimodule/{cimoduleid}")
-    public ResponseEntity<Void> addCimodule(@PathVariable int id, @PathVariable int cimoduleid) {
+    public ResponseEntity<Void> addCimodule(@PathVariable int id, @PathVariable long cimoduleid) {
```

Comment:
Ik ga er van uit dat "id" in deze methodes op een "televisionId" slaat. Dat is al wel duidelijk, omdat je in de televisionController bent. Het zou extra duidelijk zijn als je een naam als "televisionId" naast "ciModuleId" gebruikt, zodat je in een oog opslag ziet wat wat is, zonder dat je hoeft te kijken in welke klasse je bent.